src/t2iapi/biceps/codedvalue.proto

```diff
@@ -0,0 +1,37 @@
+/*
+This Source Code Form is subject to the terms of the MIT License.
+Copyright (c) 2024 Draegerwerk AG & Co. KGaA.
+
+SPDX-License-Identifier: MIT
+*/
+
+syntax = "proto3";
+
+package t2iapi.biceps;
+
+option java_package = "com.draeger.medical.t2iapi.biceps";
+option java_outer_classname = "CodedValueProto";
+
+import "t2iapi/biceps/localizedtext.proto";
+import "google/protobuf/wrappers.proto";
+
+/*
+Represents pm:CodedValue (defined in IEEE Std 11073-10207-2017).
+*/
+message CodedValueMsg {
```

Comment:
CodedValue is already implemeted, take a look if it can be used or maybe extended

Reply:
as discussed, we leave it as it is